proposals/utils/checkers.py

```diff
@@ -0,0 +1,1016 @@
+from braces.forms import UserKwargModelFormMixin
+
+from django.utils.translation import gettext as _
+from django.core.exceptions import ImproperlyConfigured
+from django.urls import reverse
+
+from proposals import forms as proposal_forms
+from studies import forms as study_forms
+from interventions import forms as intervention_forms
+from observations import forms as observation_forms
+from tasks import forms as tasks_forms
+
+from tasks.views import task_views, session_views
+from tasks.models import Task, Session
+
+from .stepper_helpers import RegularProposalLayout, PlaceholderItem, StepperItem
+
+
+class BaseStepperComponent:
+
+    def __init__(self, stepper, parent=None):
```

Comment:
Maybe just make this stuff part of the Checker and get rid of BaseStepperComponent?

Reply:
Hmm... I like `BaseStepperComponent` to keep track of what is shared between items and checkers for now.